wordpress-mobile/WordPress-iOS
# Reader Saved Posts: Add promo alert on first save

This PR adds an informative alert the first time the user tries to use the new Reader Saved Posts feature:

![simulator screen shot - iphone 8 - 2018-06-15 at 07 14 13](https://user-images.githubusercontent.com/4780/41451226-1e471ef6-706c-11e8-8962-b2042a82f9ca.png)

**To test:**

* Visit the Reader
* Save a post and ensure you are shown the promo alert above
* Try to save another post. Ensure that the promo isn't shown a second time.
